### PR TITLE
Case list sort: ability to control where blanks display

### PR DESCRIPTION
**Pull request by orangejenny**
https://manage.dimagi.com/default.asp?247874

Depends on https://github.com/dimagi/commcare-core/pull/536, will be available when mobile 2.35 is released.

@dannyroberts 

fyi @amstone326 

screen shot 2017-03-14 at 8 48 57 pm

### CODE REVIEW

**Comment by amstone326** [comment]
@orangejenny What do you think of adding a question mark hover-hint for this? Something like "Should cases for which this sort property is empty be shown at the top of the case list or the bottom?". Thinking it might not be obvious what this property does otherwise.

**Comment by orangejenny** [comment]
@amstone326 good thought, added in https://github.com/dimagi/commcare-hq/pull/15372/commits/9df33939d64da487801485f028f2fcfa7fe83ede

File: corehq/apps/app_manager/templates/app_manager/v1/partials/case_list.html [hunk 1]
```diff
@@ -90,23 +90,34 @@
...
+                                      is useful if for example you want higher
+                                      priority cases to appear closer to the
+                                      top of the list. The case list will sort
+                                      by the first property, then the second,
+                                      etc.
+                                  {% endblocktrans %}" >
+                        </span>
+                    </th>
+                    <th>{% trans "Direction" %}</th>
+                    <!-- ko if: hqImport('app_manager/js/app_manager.js').checkCommcareVersion("2.35") -->
```
Comment (dannyroberts) [inline corehq/apps/app_manager/templates/app_manager/v1/partials/case_list.html]: At some point it was best practice to put version checks for features in https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/app_manager/feature_support.py, as a way to document that feature's dependency on that version. Not sure if we abandoned that on purpose, but I still like that convention.

Comment (orangejenny) [inline corehq/apps/app_manager/templates/app_manager/v1/partials/case_list.html]: thanks, fixed!

Comment (dannyroberts) [inline corehq/apps/app_manager/templates/app_manager/v1/partials/case_list.html]: nice! Glad too that it was a really easy fix like that :)

**Review by dannyroberts — APPROVED** [review]

**Pull request merged**